lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -68,6 +79,7 @@ def to_native(self, course_overview):
             "video_outline": video_outline_url,
             "course_updates": course_updates_url,
             "course_handouts": course_handouts_url,
+            "discussion_url": discussion_url,
```

Comment:
Please also update the corresponding docstring in mobile_api/users/views.py to include the additional field.